app/ai/voice/agents/breeze_buddy/template/global_function.py

```
@@ -162,17 +334,27 @@ async def wrapper_handler(llm_args, flow_manager):
                     llm_args,
                     function_config=captured_func,
                 )
+                if captured_func.post_actions and captured_bot_instance:
+                    asyncio.create_task(
+                        _execute_post_actions_async(
+                            captured_bot_instance,
+                            result,
+                            captured_func.post_actions,
+                            captured_func.name,
+                        )
+                    )
```

Ruff (0.15.6)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)